hr_attendance_report_theoretical_time/wizards/wizard_theoretical_time.py

```diff
@@ -58,4 +60,7 @@ def view_report(self):
             'hr_attendance_theoretical_action'
         ).read()[0]
         action['domain'] = [('employee_id', 'in', self.employee_ids.ids)]
+        context = literal_eval(action['context'])
```

Comment:
Doing this will fail on some cases. You need to remove it other way, or start from a clean context. Is the previous context needed?

Reply:
IMP previous context is not needed, although it is also true that the default filters (current month or previous month) may be useful.

Reply:
Yes, they are. Then fix the context on code by a direct assignation instead of removing the expected one.